.github/workflows/push.yml

```diff
@@ -2,62 +2,38 @@ name: Push Events
 
 on:
   push:
```

Comment:
I would suggest adding the branches: - main -dev back here.

Reply:
If I'm remembering correctly, I think Scott and I removed it so that we could work in branches not named `dev` (ie, `feature/foo`). However, nothing in this action would run, so maybe it makes sense to add them back. And then, in that case, we could work in `feature/foo` and push/merge to `dev` when we're ready to deploy a test. Does that make sense to you?

Reply:
I think that makes sense. It'd be nice to align on our conventions so everything acts as we would expect when switching between projects. Scotts been trying to keep me in line for deviating in some places 😆. If the conventions aren't working we should discuss the use case.

Reply:
I may have steered you wrong in this case, @jacobdadams. I do think that push.yml should run only on pushes to main and dev. If you want the deploy on every push iteration schedule, then you can work on dev.